services/facade/serverless.yml

```diff
@@ -0,0 +1,84 @@
+service: facade
+
+plugins:
+  - serverless-webpack
+  - serverless-offline #serverless-offline needs to be last in the list
+
+custom:
+  webpackIncludeModules: true
+  eubfrEnvironment: ${env:EUBFR_ENV, 'dev'}
+  EUBFRbucketName: eubfr-${self:provider.stage}
+  serverless-offline:
+    port: 4000
+
+package:
+  individually: true
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+  stage: ${opt:stage, file(../../config.json):stage, 'dev'}
+  region: ${opt:region, file(../../config.json):region, 'eu-central-1'}
+  # Variables for .js files
+  environment:
+    REGION: ${opt:region, file(../../config.json):region, 'eu-central-1'}
+  deploymentBucket:
+    name: eubfr-${self:custom.eubfrEnvironment}-deploy
+  stackTags:
+    ENV: ${self:custom.eubfrEnvironment}
+  versionFunctions: false
+  stackPolicy:
+    - Effect: Allow
+      Principal: "*"
+      Action: "Update:*"
+      Resource: "*"
+  iamRoleStatements:
+    -  Effect: "Allow"
+       Action:
+         - "s3:*"
+       Resource: "*"
+functions:
+  upsert-projects:
+    handler: src/put.handler
+    memorySize: 128
+    events:
+      - http:
+          path: projects
+          method: put
+          private: true
+          cors: true
+  get-projects:
+    handler: src/get.handler
+    memorySize: 128
+    events:
+      - http:
+          path: projects
+          method: get
+          private: true
+          cors: true
+  get-single-project:
+    handler: src/getSingle.handler
+    memorySize: 128
+    events:
+      - http:
+          path: projects/{id}
+          method: get
+          request:
+            parameters:
+              paths:
+                id: true
+          private: true
+          cors: true
+  delete-single-project:
```

Comment:
I don't think there will ever be such a feature: "delete only this single project" ;)

Reply:
Alright, deleting this one.